src/Atmos/Model/AtmosModel.jl

```diff
@@ -252,8 +248,14 @@ end
   ν, τ = turbulence_tensors(atmos.turbulence, state, diffusive, aux, t)
   D_t = (ν isa Real ? ν : diag(ν)) * inv_Pr_turb
   d_h_tot = -D_t .* diffusive.∇h_tot
+
+  z  = altitude(atmos.orientation, aux)
+  w_sub = subsidence_velocity(atmos.subsidence, z)
+  k̂ = vertical_unit_vector(atmos.orientation, aux)
+  flux.ρe += state.ρ*w_sub .* k̂ .* diffusive.∇h_tot
```

Comment:
This needs to go not into the flux, but into the source (currently in `source.jl`): 
```suggestion
  source.ρe += state.ρ*w_sub .* k̂ .* diffusive.∇h_tot
```